src/main/scala/sangria/relay/Connection.scala

```diff
@@ -32,21 +31,25 @@ object Connection {
     case _ ⇒ false
   }
 
-  def definition[Ctx, Conn[_], Val](
+  def definition[Ctx, Conn[_, _ <: PageInfo], Val](
     name: String,
     nodeType: OutputType[Val],
+    pageInfoType: OutputType[PageInfo] = PageInfo.defaultPageInfoType,
     edgeFields: ⇒ List[Field[Ctx, Edge[Val]]] = Nil,
-    connectionFields: ⇒ List[Field[Ctx, Conn[Val]]] = Nil
-  )(implicit connEv: ConnectionLike[Conn, Val, Edge[Val]], classEv: ClassTag[Conn[Val]]) = {
-    definitionWithEdge[Ctx, Conn, Val, Edge[Val]](name, nodeType, edgeFields, connectionFields)
+    pageInfoFields: ⇒ List[Field[Ctx, PageInfo]] = Nil,
```

Comment:
`pageInfoFields` is not used anymore, isn't it?

Reply:
It was a mistaken. I fixed it.